arm/Microsoft.Storage/storageAccounts/deploy.bicep

```
@@ -71,7 +74,7 @@ param queueServices object = {}
 @description('Optional. Table service and tables to create.')
 param tableServices object = {}
 
-@description('Optional. Indicates whether public access is enabled for all blobs or containers in the storage account.')
+@description('Optional. Indicates whether public access is enabled for all blobs or containers in the storage account. For security reasons, it is recommended to set it to false.')
 param allowBlobPublicAccess bool = true
```

Should we then not also change the default value?
```suggestion
param allowBlobPublicAccess bool = false
```

---

Values in deploy.bicep and descriptions in readme file updated